walsamlee/politico
# #163416939 Add Signup Endpoint

#### What does this PR do?
Add `signup` endpoint
#### Description of Task to be completed?
Have user signup
#### How should this be manually tested?
- Clone the project, https://github.com/walsamlee/politico
- Check out into the proper branch, `ft-create-account-163416939`
#### What are the relevant pivotal tracker stories?
#163416939
